docs/readthedocs/models/input/participant/evcs.rst

```diff
@@ -1,5 +1,123 @@
+| Cee16ASocket                  | cee16                                         | 11            | AC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| Cee32ASocket                  | cee32                                         | 22            | AC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| Cee63ASocket                  | cee63                                         | 43            | AC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| ChargingStationType1          | cst1, stationtype1, cstype1                   | 7.2           | AC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| ChargingStationType2          | cst2, stationtype2, cstype2                   | 43            | AC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| ChargingStationCcsComboType1  | csccs1, csccscombo1                           | 11            | DC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| ChargingStationCcsComboType2  | csccs2, csccscombo2                           | 50            | DC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| TeslaSuperChargerV1           | tesla1, teslav1, supercharger1, supercharger  | 135           | DC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| TeslaSuperChargerV2           | tesla2, teslav2, supercharger2                | 150           | DC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+| TeslaSuperChargerV3           | tesla3, teslav3, supercharger3                | 250           | DC                    |
++-------------------------------+-----------------------------------------------+---------------+-----------------------+
+
+
+Custom Types
+""""""""""""
+While the provided standard types should be suitable for most scenarios, providing an individual type for a specific
+scenario might be necessary. To do so, a custom type can be provided instead of a common id. This custom type is tested
+against the following regex :code:`(\w+\d*)\s*\(\s*(\d+\.?\d+)\s*\|\s*(AC|DC)\s*\)`, or more generally, the custom
+type string has to be in the following syntax::
+
+    <Name>(<Apparent Power in kVA>|<AC|DC>) e.g. FastCharger(50|DC) or SlowCharger(2.5|AC)
```

Comment:
We should consider using similar typing of characteristics, don't we? In most other cases we use something, that is JSON-like. This implies no favour for either solution, but if you agree, you might hand in an issue proposing to harmonise notations.

Reply:
Agree. 
Choosing this approach over the existing ones (either having it's own type file, a JSON- or a JSON-like-String) has been motivated by the number of attributes (id, apparent power and current) which is with 3 (actual 2) very much nothing. So having it as JSON-/JSON-like string would cause a way longer string which IMHO highly decrease readability. On the other hand, having a type file with only 3 columns but the huge overhead this creates when reading and writing types I also thought this might be overkill for this small piece of information. 

tl;dr

The main reasons for choosing this approach over JSON-/JSON-like or type file are:

- better readability
- no overhead for code adaptions (file hierarchy, file types, sinks, sources, ...)
- not breaking old data models bc if missing charging station support 


I'm not sure, if harmonizing syntax makes sense at this point though. Characteristics (and I assume you refer to WEC characteristics or similar) are more or less time series with way more and different information than the type information. I'm open to discuss this and hand in an issue for harmonizing but maybe you could elaborate a little bit more why harmonizing would make sense here?!

Reply:
ack